python/tvm/_ffi/runtime_ctypes.py

```diff
@@ -240,3 +240,12 @@ class TVMArray(ctypes.Structure):
                 ("byte_offset", ctypes.c_uint64)]
 
 TVMArrayHandle = ctypes.POINTER(TVMArray)
+
+class TVMNDArrayContainer(ctypes.Structure):
```

Comment:
This definition simply discards the last two fields of `NDArray::Container` (`ref_counter_`, and `shape_`), and assumes the memory layout follows the exact order defined in ndarray.h. I am not quite sure that it is the correct way to do the ctypes like this...

Reply:
It is fine as long as the alignment assumption holds, alternatively, we can define a C API to query the array index

Reply:
Will switch to C API